### DISCUSSION TITLE
Export a Docker Compose Project using 'podman generate'

### DISCUSSION THREAD

**Discussion by djnotes** [containers/podman#12695] [Q&A]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind question

**Description**

I came across a seemingly simple idea of exporting my WordPress project that was running using Docker Compose to a kubernetes description file using `podman generate`. It seemed simple, but then I found I could not reuse the existing containers to add them to a pod in order to export them later. At least, I didn't know how to. To the extent of my understanding, I have to recreate the containers with similar settings as in the Compose file and add them to a pod. Once the pod is up and running with the containers in it, I can use `podman generate` to export the containers to a kube.yaml file that can be played later with `podman play`.   
Is this the only way? Is it possible to more easily add services defined in a Docker Compose project to a pod in order to take advantage of Podman's kubernetes file generation and replay?

**Comment by vrothberg** [comment]
@baude PTAL

**Comment by rhatdan** [comment]
Would this be better to be handled with https://kompose.io/?
